src/Marten/Linq/MartenLinqQueryable.cs

```diff
@@ -47,6 +47,9 @@ public MartenLinqQueryable(IMartenSession session): base(new MartenLinqQueryProv
 
         public MartenLinqQueryProvider MartenProvider { get; }
 
+        // TODO -- Convert this to property?
```

Comment:
Wasn't sure if I should leave this `TODO` here for later, so updated it to match the style of other `TODOs` in the code.

Reply:
That's fine as is.